imagedata/image_data.go

```diff
@@ -79,22 +80,25 @@ func loadWatermark() (err error) {
 }
 
 func loadFallbackImage() (err error) {
-	if len(config.FallbackImageData) > 0 {
+	switch {
+	case len(config.FallbackImageData) > 0:
 		FallbackImage, err = FromBase64(config.FallbackImageData, "fallback image")
-		return
-	}
-
-	if len(config.FallbackImagePath) > 0 {
+	case len(config.FallbackImagePath) > 0:
 		FallbackImage, err = FromFile(config.FallbackImagePath, "fallback image")
-		return
+	case len(config.FallbackImageURL) > 0:
+		FallbackImage, err = Download(config.FallbackImageURL, "fallback image", nil, nil)
+	default:
+		FallbackImage, err = nil, nil
 	}
 
-	if len(config.FallbackImageURL) > 0 {
-		FallbackImage, err = Download(config.FallbackImageURL, "fallback image", nil, nil)
-		return
+	if FallbackImage != nil && err == nil && config.FallbackTTL > 0 {
+		if FallbackImage.Headers == nil {
+			FallbackImage.Headers = make(map[string]string)
+		}
+		FallbackImage.Headers["FallbackTTL"] = strconv.Itoa(config.FallbackTTL)
```

Comment:
We don't need to actually store the fallback TTL here as we can get it from the config. It's enough just to mark the `FallbackImage` with `FallbackImage.Headers["FallbackImage"] = "1"`.

Reply:
```suggestion
		FallbackImage.Headers["Fallback-Image"] = "1"
```